src/createActionPack.js

```diff
@@ -0,0 +1,19 @@
+import isFunction from 'lodash/isFunction'
```

Comment:
This isn’t needed, please use `typeof === ‘function’` instead of bringing in a massive dependency.